tests/pw/.env.bak

```
@@ -0,0 +1,36 @@
+DB_PORT=9998
+DB_PREFIX=wp
+
+# REST API (auto-detected by setup)
+SERVER_URL=http://localhost:9999/?rest_route=
+
+
+SERVER_URL=http://localhost:9999/wp-json
+
+SERVER_URL=http://localhost:9999/wp-json
+SERVER_URL=http://localhost:9999/wp-json
+SERVER_URL=http://localhost:9999/wp-json
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The SERVER_URL key is duplicated

(DuplicatedKey)